.gitignore

```diff
@@ -3,4 +3,3 @@ node_modules
 
 # Generated
 dist
-index.d.ts
```

Comment:
I moved the type declaration to the `dist` folder because that's where all the build assets are, I don't have a strong opinion on this, it just makes sense to me.

Reply:
Yeah, that seems like a better place for it 👍